### PR TITLE
Updates to work with SAM pull request 1325 and ssc pull request 969

### PR DESCRIPTION
**Pull request by sjanzou**
Updates for pysam to work with flattened SAM UI variables (e.g. Adjustment Factors)

### CODE REVIEW

File: build_manylinux_arm64.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ do
    retVal=$?
    if [ $retVal -ne 0 ]; then
        echo "Error in Tests"
-       exit 1
+#       exit 1
```
Comment (dguittet) [inline build_manylinux_arm64.sh:31]: please revert

File: build_conda_arm64.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ cd $PYSAMDIR || exit
 source $(conda info --base)/etc/profile.d/conda.sh
 
 # get `CONDA_TOKEN` env variable, required for authentication
-source ~/.env || exit
+#source ~/.env || exit
```
Comment (dguittet) [inline build_conda_arm64.sh:7]: we could remove the `|| exit` part?

**Review by dguittet — APPROVED** [review]
Looks great! Just some minor changes then I think it's good to merge

File: build_conda_arm64.sh (hunk 2 above)
Comment (sjanzou) [reply build_conda_arm64.sh]: Updated in [67aa4b5](https://github.com/NREL/pysam/pull/146/commits/67aa4b57c75bd549f0b6ce8c1a498c013456ce8e)

File: build_manylinux_arm64.sh (hunk 1 above)
Comment (sjanzou) [reply build_manylinux_arm64.sh]: Updated in [67aa4b5](https://github.com/NREL/pysam/pull/146/commits/67aa4b57c75bd549f0b6ce8c1a498c013456ce8e)

**Pull request merged**